docs/algolia-search-indexing.md

```
@@ -1,79 +1,56 @@
 
-This document describes how Algolia search indexing is configured for the atmos.tools documentation site.
+This document describes how Algolia search indexing is configured for the `atmos.tools` documentation site.
 
 ## Overview
 
-The atmos.tools documentation uses [Algolia DocSearch](https://docsearch.algolia.com/) for search functionality. Search indexing is managed via the **Algolia Crawler**, which is triggered automatically on every deployment.
+The `atmos.tools` documentation uses [Algolia DocSearch](https://docsearch.algolia.com/) for search. Search records are produced by the Algolia Crawler. The crawler configuration is managed in this repository, validated on pull requests, and deployed only after changes merge to `main`.
 
 ## Architecture
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)